tests/cugraph/cugraph-ops/test_cugraph_relgraphconv.py

```diff
@@ -1,97 +1,77 @@
+# pylint: disable=too-many-arguments, too-many-locals
+from collections import OrderedDict
+from itertools import product
+
+import dgl
 import pytest
 import torch
-import dgl
-from dgl.nn import CuGraphRelGraphConv
-from dgl.nn import RelGraphConv
+from dgl.nn import CuGraphRelGraphConv, RelGraphConv
 
 # TODO(tingyu66): Re-enable the following tests after updating cuGraph CI image.
```

Comment:
Is this now re-enabled?

Reply:
Not yet, I will create a new image after our 23.02 release and update these pytest markers in a separate PR.